packages/react-server-integration-tests/gulpfile.babel.js

```diff
@@ -33,8 +34,12 @@ function getSpecGlob (prefix) {
 	return specs;
 }
 gulp.task("compile", () => {
-	return gulp.src("src/**/*.js")
-		.pipe(babel())
+	const jsFilter = filter("**/*.js");
+
+	return gulp.src("src/**")
+		.pipe(jsFilter)
+			.pipe(babel())
+		.pipe(jsFilter.restore())
```

Comment:
The tests were only copying over JS files in the build, but they need to copy over CSS/images/fonts too.